webapp/models.py

```diff
@@ -0,0 +1,41 @@
+from sqlalchemy import Column, Date, Integer, String, DateTime
+from sqlalchemy.orm import DeclarativeBase
+from sqlalchemy.sql import func
+
+class Base(DeclarativeBase):
+  pass
+
+
+class PreviousSession(Base):
+  __tablename__ = "previous_sessions"
+
+  id = Column(Integer, primary_key=True)
+  topic = Column(String, nullable=False)
+  owner = Column(String, nullable=False)
+  duration = Column(String, nullable=False)
+  date = Column(Date, nullable=False, default=func.now())
+  slides = Column(String, nullable=True)
+  recording = Column(String, nullable=False)
+  description = Column(String, nullable=True)
+  chat_log = Column(String, nullable=True)
+  tags = Column(String, nullable=True)
```

Comment:
might be too much effort for this project since the db schema is quite simple, but perhaps we could split out things like tags into a separate table with defined foreign key constraints? Just a suggestion though

Reply:
I don't want to add optimize the data model in this PR, I want to keep it simple and readable for this PR, but I think it could be a good next step.